shared/js/content-scope/protections.js

```diff
@@ -29,8 +29,7 @@ export async function loadProtections () {
         'gpc',
         'hardware',
         'referrer',
-        'screen-size',
-        'temporary-storage'
```

Comment:
This is in Chrome :) We should leave it as is until Chrome completely removes it.

![webkit-chrome](https://user-images.githubusercontent.com/985504/124769079-7b517d00-df39-11eb-8db1-216eb276b374.jpg)

Reply:
Ah good to know!